.github/workflows/go.yml

```diff
@@ -78,8 +81,19 @@ jobs:
           path: source
           fetch-depth: 0
 
+      - name: Setup Go
+        uses: FerretDB/github-actions/setup-go@main
+        with:
+          cache-key: fuzz
+
+      - name: Install Task
+        run: |
+          cd tools
+          go generate -x
```

Comment:
Why not `cd tools; go generate -x` as in other steps?

Reply:
Fixed